.gitlab-ci.yml

```diff
@@ -0,0 +1,27 @@
+stages:
```

Comment:
What's the reason for adding gitlab-CI? Is there a mirror I'm unaware of?

Reply:
@sgeisler This was used to build on our server. but maybe someone will need in the future

Reply:
Ok, could it run on any other gitlab instance given that it references your docker registry? Is your linux image public?

Reply:
Our image is public, but it is better to use the image with [rust stable](https://hub.docker.com/_/rust) from dockerhub